content/en/docs/aws/troubleshooting-aws.md

```diff
@@ -132,6 +132,49 @@ ks delete default -c ${COMPONENT}
 ks param set ${COMPONENT} dnsName fs-0xxxxx2a216cf.fsx.us-west-2.amazonaws.com
 ks apply default -c ${COMPONENT}
 ```
+### Amazon RDS Connectivity Issues
+
+If you run into CloudFormation deployment errors, you can use [troubleshooting guide](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/troubleshooting.html) to find a resolution.
+
+If you have connectivity issues with Amazon RDS, try launching mysql-client container and connecting to your RDS endpoint. This will let you know if you have network connectivity with the db and also if db is created properly
```

Comment:
Minor suggestion: please add a period(.) at the end of the sentence.

Reply:
resolved